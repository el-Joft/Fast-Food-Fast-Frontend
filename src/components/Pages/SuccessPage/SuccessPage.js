import React from 'react';

const SuccessPage = () => (
  <section>
    <div className="successful">
      <div>
        <img src="/public/images/good.jpg" alt="success" />
      </div>
      <div className="success-text">
        <h3>Successfully made an Order</h3>
        <p>
You can check your
          {' '}
          <a href="user-dashboard.html">dashboard</a>
        </p>
      </div>
    </div>

  </section>
);

export default SuccessPage;
