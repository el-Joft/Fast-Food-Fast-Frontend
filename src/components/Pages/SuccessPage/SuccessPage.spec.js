/* eslint-disable react/react-in-jsx-scope */
import { shallow } from 'enzyme';
import { SuccessPage } from './SuccessPage';

describe('rendering', () => {
  let wrapper;
  it('should render the component', () => {
    wrapper = shallow(<SuccessPage />);
    expect(wrapper.length).toBe(1);
  });

  it('should render the component', () => {
    wrapper = shallow(<SuccessPage />);
    const section = wrapper.find('section');
    expect(section.length).toBe(1);
  });

  it('should render the component', () => {
    wrapper = shallow(<SuccessPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
