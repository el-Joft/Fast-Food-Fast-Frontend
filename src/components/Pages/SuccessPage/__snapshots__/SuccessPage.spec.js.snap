// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering should render the component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SuccessPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="successful"
      >
        <div>
          <img
            alt="success"
            src="/src/public/images/good.jpg"
          />
        </div>
        <div
          className="success-text"
        >
          <h3>
            Successfully made an Order
          </h3>
          <p>
            You can check your
             
            <a
              href="user-dashboard.html"
            >
              dashboard
            </a>
          </p>
        </div>
      </div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div>
            <img
              alt="success"
              src="/src/public/images/good.jpg"
            />
          </div>,
          <div
            className="success-text"
          >
            <h3>
              Successfully made an Order
            </h3>
            <p>
              You can check your
               
              <a
                href="user-dashboard.html"
              >
                dashboard
              </a>
            </p>
          </div>,
        ],
        "className": "successful",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              alt="success"
              src="/src/public/images/good.jpg"
            />,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "success",
              "src": "/src/public/images/good.jpg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Successfully made an Order
              </h3>,
              <p>
                You can check your
                 
                <a
                  href="user-dashboard.html"
                >
                  dashboard
                </a>
              </p>,
            ],
            "className": "success-text",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Successfully made an Order",
              },
              "ref": null,
              "rendered": "Successfully made an Order",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "You can check your",
                  " ",
                  <a
                    href="user-dashboard.html"
                  >
                    dashboard
                  </a>,
                ],
              },
              "ref": null,
              "rendered": Array [
                "You can check your",
                " ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "dashboard",
                    "href": "user-dashboard.html",
                  },
                  "ref": null,
                  "rendered": "dashboard",
                  "type": "a",
                },
              ],
              "type": "p",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="successful"
        >
          <div>
            <img
              alt="success"
              src="/src/public/images/good.jpg"
            />
          </div>
          <div
            className="success-text"
          >
            <h3>
              Successfully made an Order
            </h3>
            <p>
              You can check your
               
              <a
                href="user-dashboard.html"
              >
                dashboard
              </a>
            </p>
          </div>
        </div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              <img
                alt="success"
                src="/src/public/images/good.jpg"
              />
            </div>,
            <div
              className="success-text"
            >
              <h3>
                Successfully made an Order
              </h3>
              <p>
                You can check your
                 
                <a
                  href="user-dashboard.html"
                >
                  dashboard
                </a>
              </p>
            </div>,
          ],
          "className": "successful",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="success"
                src="/src/public/images/good.jpg"
              />,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "success",
                "src": "/src/public/images/good.jpg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3>
                  Successfully made an Order
                </h3>,
                <p>
                  You can check your
                   
                  <a
                    href="user-dashboard.html"
                  >
                    dashboard
                  </a>
                </p>,
              ],
              "className": "success-text",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Successfully made an Order",
                },
                "ref": null,
                "rendered": "Successfully made an Order",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    "You can check your",
                    " ",
                    <a
                      href="user-dashboard.html"
                    >
                      dashboard
                    </a>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  "You can check your",
                  " ",
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "dashboard",
                      "href": "user-dashboard.html",
                    },
                    "ref": null,
                    "rendered": "dashboard",
                    "type": "a",
                  },
                ],
                "type": "p",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
